query.py

```diff
@@ -0,0 +1,21 @@
+import mysql.connector
+
+def script(s):
+    try:
+        db = mysql.connector.connect(user='', password='',#user & #password
+                                     host='127.0.0.1', database="bikepoint")
+        cursor = db.cursor()
+        querry='SELECT * FROM bikepoint.point where commonName like \'%{query}%\';'
```

Comment:
Avoid using incorrect and ambiguous spellings - creates space for very difficult to find problems later